Subject: [PATCH] fbdev: Do not let CONFIG_FB_DEFERRED_IO default to y

From: Geert Uytterhoeven <geert@linux-m68k.org>

CONFIG_FB_DEFERRED_IO can not be turned off, while it's already selected
automatically by the drivers that need it.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/video/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -147,7 +147,7 @@ config FB_SYS_FOPS
 config FB_DEFERRED_IO
 	bool
 	depends on FB
-	default y
+	default n
 
 config FB_SVGALIB
 	tristate
